Source/Scene/GoogleEarthEnterpriseImageryProvider.js

```diff
@@ -229,7 +229,7 @@ function GoogleEarthEnterpriseImageryProvider(options) {
         const e = new RuntimeError(
           `The server ${metadata.url} doesn't have imagery`
         );
-        metadataError = TileProviderError.handleError(
+        metadataError = TileProviderError.reportError(
```

Comment:
So this was previously not passing the correct paramaters to the old `handleError` correctly, right? There should have been an additional `undefined` for the retryCallback?